app/views/management/budgets/print_investments.html.erb

```diff
@@ -1,12 +1,10 @@
 <table>
-<% @budgets.each do |budget| %>
-  <tr id="<%= dom_id(budget) %>">
-    <td><%= budget.name %></td>
-    <td><%= budget.translated_phase %></td>
+  <tr id="<%= dom_id(@budget) %>">
```

Comment:
I think we're not covering the scenario where @budget may not be present (for new clean installations without a proper database seed, or that have cleaned up "dummy data" from seeds)

Reply:
Yes :) +1 to an issue for this 👌 
These are the tests that where failing due to not having a budget, it would be a great starting point to implementa that issue 👍 
https://github.com/consul/consul/pull/2322/commits/34e0c23bb37bbd0c7b9b17d26a71d6adec9802e2